pages/[row].js

```diff
@@ -40,15 +36,19 @@ export async function getStaticPaths() {
 }
 
 export async function getStaticProps({ params: { row } }) {
-  const response = await sheets.spreadsheets.values.get({
-    spreadsheetId: process.env.SHEET_ID,
-    range: `Organizations (English)!${parseInt(row) + 1}:${parseInt(row) + 1}`,
```

Comment:
As far as I can tell, this fetches the entire spreadsheet and filters after the fact. If this is not correct i can revert this refactor for efficiency